code/game/objects/structures/flora.dm

```diff
@@ -358,4 +358,16 @@
 	icon_state = "bushb"
 
 /obj/structure/flora/junglebush/c
-	icon_state = "bushc"
\ No newline at end of file
+	icon_state = "bushc"
+
+/obj/structure/flora/junglebush/large
+	icon_state = "bush"
+	icon = 'icons/obj/flora/largejungleflora.dmi'
+	pixel_x = -16
+	layer = FLY_LAYER

```

Comment:
ABOVE_ALL_MOB_LAYER
Or WALL_OBJ_LAYER, if you want large mobs to appear above these.